.k8s.mk

```diff
@@ -41,7 +41,7 @@ include .vagrant.mk
 include .packet.mk
 
 # .null.mk allows you to skip the vagrant machinery with:
-# export CLUSTER_RULES_PREFIX=null
+ export CLUSTER_RULES_PREFIX=null
```

Comment:
I don't think we want to default to this. Let's stick with Vagrant for the time being.

Reply:
Yes, but for VM were nested virtualization can not be authorized, vagrant can not be used to create VM. That was the case for me when I started the set up of NSM, deploying vagrant VMs was not possible. That is way I was creating an other k8s cluster in tow separate VMs, and then deploy NSM on the master.
I think it is a novel way more generic to deploy NSM without using vagrant.

Reply:
Well, this particular line just changes the default `CLUSTER_RULES_PREFIX` from `vagrant` to `null`. We would prefer to keep `vagrant` as the default for now and allow interested people to explicitly `export CLUSTER_RULES_PREFIX=null` if they need to.

Reply:
Ok, I get it!